src/main/cpp/t08_cycleswap.cpp

```diff
@@ -19,5 +19,24 @@
 using namespace std;
 
 int t08_cycleswap() {
+	int n, b, z;
+	b = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'b' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4436470)